srcpkgs/crun/template

```diff
@@ -25,13 +27,6 @@ case "$XBPS_MACHINE" in
 	*) ;;
 esac
```

Comment:
I don't think this is necessary, we are only matching architecture on the host machine, mips* is missing and we will just forget to add more archs when we add them.

Reply:
I guess this was added for the same reason as this: https://github.com/void-linux/void-packages/commit/61ef978
Maybe @q66 can comment here